test/integration_test_helper.rb

```diff
@@ -15,6 +15,8 @@
   Rails.logger.error("DO NOT IGNORE - Consistent failure - #{klass} #{test_name}")
 end
 
+Capybara.server = :webrick
```

Comment:
This was the default for 2.x. Puma is the default for 3.x but I'd rather want to discuss this and handle it an separate issue.